src/app/work-item/work-item-board/work-item-board.component.ts

```diff
@@ -353,8 +354,35 @@ export class WorkItemBoardComponent implements OnInit, OnDestroy {
   }
 
   onMoveToBacklog(event: any): void {
-    alert('Not Implemented yet');
     event.stopPropagation();
+    //set this work item's iteration to None
+    //send a patch request
+    this.workItem.relationships.iteration = {}
```

Comment:
So, are we going to unassociated the iteration from the WI or associate it with root iteration?